transform/binary-plugin/binary_plugin.go

```diff
@@ -30,8 +32,10 @@ func (b *BinaryPlugin) Run(u *unstructured.Unstructured) (transform.PluginRespon
 	}
 
 	if len(errBytes) != 0 {
-		b.log.Errorf("error from plugin binary")
-		return p, fmt.Errorf("error from plugin binary: %s", string(errBytes))
+		logs = strings.Split(string(errBytes), "\n")
+		for _, line := range logs {
+			b.log.Info("Plugin Log line: ", line)
```

Comment:
Probably want make this debug, I don't think we want to print this everytime there is an error.

Reply:
Would it be overkill to set an expected format for plugin stderr lines to allow plugins to log info/error-level output? The default could then be made debug, but if there was something the plugin thought was important for the end user to see, they could indicate a more prominent log level by starting the lines with something like "INFO:" or "ERROR:". Then this code could alter its log statement based on the line's prefix.

Reply:
it could do that I agree, parsing lines for context and then making decisions always seems to be a potential source of problems IMO. 

I think that another thing to consider is that when plugins are working, I don't think the user cares about them, just the output. If there is something going wrong is when we need to get the extra info. I think in that case it probably makes sense to move these to only debug. 

Does increase the need for konveyor/crane#7

Reply:
OK, that makes sense. Also, I'm realizing that given the isolated of the transform CLI, if there's an error, it's easy enough to re-run with debug on if errors happen. It's not like crane 1.x where you can't just re-run the error-triggering step with an increased debug level.

Reply:
updated. now logs at debug level